WebService/WebService.csproj

```diff
@@ -36,12 +36,12 @@
         </Content>
     </ItemGroup>
     <ItemGroup>
-        <PackageReference Include="Autofac" Version="4.9.4" />
-        <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="5.0.1" />
+        <PackageReference Include="Autofac" Version="5.1.2" />
+        <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="6.0.0" />
         <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0" />
         <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
-        <PackageReference Include="Microsoft.Extensions.Configuration.Ini" Version="3.1.0" />
-        <PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="5.6.0" />
+        <PackageReference Include="Microsoft.Extensions.Configuration.Ini" Version="3.1.3" />
+        <PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="6.5.0" />
```

Comment:
How did you test these changes, since it is a major version change (and potentially a breaking change)? (related to above comment)

Reply:
Tested by building the image and deploying it live, then running some simulations. All unit/functional tests also passed after changes were made.